pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -2015,7 +2015,7 @@ private void handleLocalPoliciesUpdates(NamespaceName namespace) {
                         if (namespace.includes(TopicName.get(name))) {
                             // If the topic is already created, immediately apply the updated policies, otherwise
                             // once the topic is created it'll apply the policies update
-                            topicFuture.thenAccept(topic -> {
+                            topicFuture.thenAcceptAsync(topic -> {
```

Comment:
Why we need to switch the thread again here? In most case the topicFuture is completed, if the topicFuture is completed, we'd better make it execute in current thread.